### DISCUSSION TITLE
为什么启用插件后，会在所有网页中植入googletagmanager的JS？

### DISCUSSION THREAD

**Discussion by shadu120** [nextai-translator/nextai-translator#667] [Q&A]
启用该插件后，所有我的开的网页都会被植入JS
`<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-D7054DX333"></script>`
如果这不是插件工作的必须方式，请问为什么要这么做？

<img width="1468" alt="image" src="https://user-images.githubusercontent.com/3023471/235909991-a186c3da-b3b2-43c0-9585-a96c562986e0.png">

**Comment by yetone** [comment]
这个问题已解决：https://github.com/yetone/openai-translator/issues/665

**Reply by shadu120** [reply]
@yetone 谢谢。请问何时推送到Chrome Web Store生效，当前版本问题依旧。

**Reply by yetone** [reply]
@shadu120 今天会发布新的版本，抱歉给您带来了困扰
